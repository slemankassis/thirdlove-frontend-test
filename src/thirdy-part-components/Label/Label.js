const React = require('react');


class Label extends React.Component {
  render() {
    return (
      1
    );
  }
}

module.exports = Label;
