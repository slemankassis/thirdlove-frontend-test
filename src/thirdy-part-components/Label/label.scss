.label{
  margin:  5px 0;
  font-family: OpenSans;
  font-size: 13px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: 1.5px;
  color: #222222;
}
