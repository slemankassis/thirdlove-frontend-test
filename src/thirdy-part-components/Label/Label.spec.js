import React from 'react';
import Label from './Label';

describe('Label', () => {
  it('renders snapshot without Props', () => {
    const component = shallow(<Label />);
    expect(escapeSnapshot(component)).toMatchSnapshot();
  });

  it('renders snapshot with fullProps', () => {
    const fullProps = {
      text: '__TEXT__',
      className: '__CLASS_NAME__',
    };

    const component = shallow(<Label {...fullProps} />);
    expect(escapeSnapshot(component)).toMatchSnapshot();
  });
});
