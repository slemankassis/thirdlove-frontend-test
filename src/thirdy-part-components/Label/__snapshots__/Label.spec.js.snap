// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders snapshot with fullProps 1`] = `
"<div className='label __CLASS_NAME__'>
  __TEXT__
</div>"
`;

exports[`Label renders snapshot without Props 1`] = `"<div className='label' />"`;
